google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -60,15 +84,10 @@ public String getGoogleUsername() {
   }
 
   @Attribute(ENVIRONMENT_ATTRIBUTE)
-  public String getEnvironment() {
+  public AppEngineEnvironment getEnvironment() {
```

Comment:
Have you tested how this affects plugin migration? I.e users that have run confine set up with older versions